src/v2/Apps/Algolia/routes/AlgoliaResults/Components/AlgoliaPagination.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Pagination } from "@artsy/palette"
+import { useScrollTo } from "v2/Utils/Hooks/useScrollTo"
+import { ANCHOR_CONTAINER_ID } from "v2/Apps/Algolia/constants"
+import { createPageCursors } from "v2/Apps/Algolia/Utils/pagination"
+
+export const AlgoliaPagination = props => {
+  const { nbPages, currentRefinement, refine } = props
```

Comment:
What does `nb` mean? what does `refine` mean? If possible we should make these variables a bit more explicit.

Reply:
* `nbPages` - The total number of existing pages.
* `refine` - Selects a specific page

You can read more about these [here](https://www.algolia.com/doc/api-reference/widgets/pagination/react/#create-a-react-component)